### PR TITLE
Added Enrollment Interface

### LINKED ISSUE #43: Create Enrollment Interface

### PR DESCRIPTION
**Pull request by Jismatov895**

### CODE REVIEW

File: src/OneFit.Service/Interfaces/Enrollments/IEnrollmentService.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using OneFit.Service.DTOs.Enrollments;
+
+namespace OneFit.Service.Interfaces.Enrollments
+{
+    public interface IEnrollmentService
+    {
+        Task<EnrollmentViewModel> CreateAsync(EnrollmentCreateModel issue);
```
Comment (makhammadsoliyev) [inline src/OneFit.Service/Interfaces/Enrollments/IEnrollmentService.cs]: Change variable name

**Review by makhammadsoliyev — CHANGES_REQUESTED** [review]
Change

**Comment by Jismatov895** [comment]
@makhammadsoliyev I have change your request

**Review by makhammadsoliyev — APPROVED** [review]
Good

**Pull request merged**